helm/superset/README.md

```diff
@@ -23,7 +23,7 @@ NOTE: This file is generated by helm-docs: https://github.com/norwoodj/helm-docs
 
 # superset
 
-![Version: 0.8.5](https://img.shields.io/badge/Version-0.8.5-informational?style=flat-square)
+![Version: 0.8.6](https://img.shields.io/badge/Version-0.8.6-informational?style=flat-square) 
```

Comment:
@rajan123456 trailing-whitespace is failing. Pls, remove the last whitespace for this line.

Reply:
Updated. Thanks!